control-plane/pkg/reconciler/broker/broker_test.go

```diff
@@ -1970,6 +1995,7 @@ func brokerFinalization(t *testing.T, format string, env config.Env) {
 			Name: "Reconciled normal - no broker found in config map",
 			Objects: []runtime.Object{
 				NewDeletedBroker(),
+				BrokerConfig(bootstrapServers, 20, 5),
```

Comment:
Are there any tests that make sure the config is actually read from the configmap?

Reply:
Yes, all of them, and there is a test that fails because we can't find the ConfigMap.